### PR TITLE
feat(reporter): update session instrument panel

### LINKED ISSUE #21400: [sessions] Enhance Sessions Instrument Panel
### What would you like?

- Design of how presented
- Alignment with header
- log: false - how does it interact? https://github.com/cypress-io/cypress/issues/21377
- Interactivity - the panel does not look interactive
- Clear one by one? Probably out of scope but see user interviews

### Designs
[Instrumentation Panel Design](https://www.figma.com/file/E86dG4e1dcuyRe33B2j3zU/app-reporter%3A-sessions?node-id=17425%3A148109)

### PR DESCRIPTION
**Pull request by emilyrohrbough**
- Closes #21400
- Apart of #17710

Update Session's Instrument Panel to match new designs: https://www.figma.com/file/E86dG4e1dcuyRe33B2j3zU/app-reporter%3A-sessions?node-id=17425%3A148109. 

This work is broken out / preps for when Global Sessions are enabled.

### User facing changelog

- UI updates were made to the Sessions Instrument Panel in the Cypress Reporter. Addresses [#21400](https://github.com/cypress-io/cypress/issues#21400).

### How has the user experience changed?

See Percy Screenshots for additional visuals.

Screen Shot 2022-08-29 at 11 06 59 AM

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [TODO] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/driver/src/cy/commands/sessions/index.ts [hunk 1]
```diff
@@ -114,6 +113,16 @@ export default function (Commands, Cypress, cy) {
         }
       }
 
+      function setSessionInfo (status: string) {
```
Comment (AtofStryker) [inline packages/driver/src/cy/commands/sessions/index.ts:116]:
```suggestion
      function setSessionLogStatus (status: string) {
```

File: packages/graphql/schemas/cloud.graphql [hunk 2]
```diff
@@ -668,10 +668,14 @@ type Mutation {
   Create a project in the dashboard and return its object
   """
   cloudProjectCreate(
+    campaign: String
     ciProviders: [String!]
+    cohort: String
```
Comment (AtofStryker) [inline packages/graphql/schemas/cloud.graphql:673]: did you change these types or were they not generated and checked in and are just showing up here?

File: packages/reporter/src/attempts/attempt-model.ts [hunk 3]
```diff
@@ -180,7 +190,17 @@ export default class Attempt {
   _addSession (props: SessionProps) {
     const session = new Session(props)
 
-    this.sessions[props.sessionInfo.id] = session
+    this.sessions[props.id] = session
+  }
+
+  _updateSession (props: SessionProps) {
```
Comment (AtofStryker) [inline packages/reporter/src/attempts/attempt-model.ts:196]:
```suggestion
  _updateOrAddSession (props: SessionProps) {
```

File: packages/reporter/src/sessions/sessions-model.ts [hunk 4]
```diff
@@ -7,19 +7,29 @@ export interface SessionProps extends InstrumentProps {
...
 export default class Session extends Instrument {
   @observable name: string
-  @observable data: SessionProps['sessionInfo']['data']
+  @observable status: string
+  @observable isGlobalSession: boolean = false
 
   constructor (props: SessionProps) {
     super(props)
-    const { id, data } = props.sessionInfo
+    const { sessionInfo } = props
```
Comment (AtofStryker) [inline packages/reporter/src/sessions/sessions-model.ts:22]: is this type safe?
```suggestion
    const { sessionInfo: { isGlobalSession, name, status } }= props
```

File: packages/graphql/schemas/cloud.graphql (hunk 2 above)
Comment (emilyrohrbough) [reply packages/graphql/schemas/cloud.graphql:673]: this is from running `yarn build`. I think this is meant to update automatically when the cloud-schema updates 🤷🏻‍♀️

File: packages/reporter/src/attempts/attempt-model.ts [hunk 5]
```diff
@@ -180,7 +190,17 @@ export default class Attempt {
...
+    const session = this.sessions[props.id]
+
+    if (session) {
+      return session.update(props)
```
Comment (AtofStryker) [inline packages/reporter/src/attempts/attempt-model.ts]: wonder if it makes sense to return after the update for some reason the signature in `Session` changes, which also might imply that `_updateSession` is always a void return

**Review by AtofStryker — APPROVED** [review]

File: packages/reporter/src/sessions/sessions-model.ts (hunk 4 above)
Comment (emilyrohrbough) [reply packages/reporter/src/sessions/sessions-model.ts]: Yes, but at the mercy of emitted log properties.  Updates seems good though 👍🏻

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**